.github/docker-compose.yml

```diff
@@ -1,18 +1,12 @@
-version: '3.9'
-
 services:
-  mysql:
-    image: mysql:5.7
+  mariadb:
+    image: mariadb:10.9
```

Comment:
I don't what the main differences are between MariaDB and MySQL, but as long as we still support MySQL (at least for now), I would prefer if we actually tested using MySQL.

Reply:
But to unblock the CI we can merge this and revisit